### DISCUSSION TITLE
Inquiry related to various pilot assignment in resource_grid

### DISCUSSION THREAD

**Discussion by hssam1002** [NVlabs/sionna#644] [Q&A]
Dear, Sionna team.

Thanks for providing Sionna as a tool for advancing wireless communication technology.
I working to develope my own simulator using Sionna.

I am wondering if it is possible to use more than one type of pilot signal, such as PDSCH DMRS and TRS, in the same resource grid.
I considered creating a unique pilot pattern to accommodate both, but I am concerned that this may cause unintered channel estimation.

Currently, I am trying to build 2 different types of resource grids and pass them into one same channel.
However, I'm worried is the SINR might differ due to normalization effects.

Is there a recommende way to incorporate both pilot signals into a single resource grid, and operate seperate channel estimation?

**Comment by jhoydis** [comment]
Hi @hssam1002,

Your question is very specific to the problem that you are working an and we cannot provide help on such a detailed level.

Out-of-the-box, Sionna works with a single pilot pattern per stream. You can fill the reserved resource elements with essentially whatever you want. However, the channel estimator uses by-default all of the symbols in a pilot pattern for channel estimation, and this is probably not what you want. But nothing prevents you from implementing your own estimator.
